01-filedir.md

```diff
@@ -348,6 +348,45 @@ it also displays another special directory that's just called `.`,
 which means "the current working directory".
 It may seem redundant to have a name for it,
 but we'll see some uses for it soon.
+
+`cd` also allows you switch back and forth between places in a rapid way,
+using `cd -`.
+Much like `..`, `-` is a special instruction that `cd` translates to
```

Comment:
Could we drop "Much like `..`" because "if `-` is like `..` I should be allow to run `cd -/some-dir` as the same way of `cd ../some-dir`".